package.json

```diff
@@ -29,7 +29,7 @@
     "doc": "npm run build:doc --workspace website",
     "changeset": "changeset add",
     "postinstall": "npm run build",
-    "lint": "eslint --cache --cache-location=node_modules/.cache/ './**/*.{js,jsx,ts,tsx,mjs}'",
+    "lint": "eslint --fix --cache --cache-location=node_modules/.cache/ './**/*.{js,jsx,ts,tsx,mjs}'",
```

Comment:
this command is also called as part of travis CI, the fix has to be either removed, or moved to a separated command